internal/monitors/etcd/metadata.yaml

```diff
@@ -0,0 +1,867 @@
+monitors:
+- monitorType: etcd
+  dimensions:
+  doc: |-
+    This monitor reports etcd server metrics under the `/metrics` path on its
```

Comment:
Discuss the difference between this and `collectd/etcd` and also give a sample config for a typical K8s env.

Reply:
Added a comment regarding the differences between the monitors and also provided a sample config